radio/src/dmafifo.h

```diff
@@ -45,6 +45,9 @@ class DMAFifo
 
     bool isEmpty()
     {
+#if defined(SIMU)
+      return true;

```

Comment:
We actually need better fix in order to be able to fill telemetry fifo with telemetry simulator in Companion! My fix was just to make it compile and run.